frontend/src/scenes/experiments/experimentsLogic.tsx

```diff
@@ -16,6 +19,17 @@ export const experimentsLogic = kea<experimentsLogicType>({
                     const response = await api.get(`api/projects/${values.currentTeamId}/experiments`)
                     return response.results as Experiment[]
                 },
+                deleteExperiment: async (id: number) => {
```

Comment:
is this called anywhere right now?

Reply:
noope, didn't know where to fit it on the table because I wanted it to do a popup confirmation , but the popup confirmation doesn't work inside of the `...` actions dropdown, so I put it in a column with a trash can icon but took it out since it's not vital right now 

Also I wasn't sure if deleting an experiment would delete its associated feature flag?